### DISCUSSION TITLE
Motifs similarity

### DISCUSSION THREAD

**Discussion by laurajohanne** [aertslab/scenicplus#327] [Q&A]
Hi All,

Thank you again for this nice package.

I am interested in during a motifs similarity analysis of some of the metacluster motifs, but since there are two PWM assigned to each metacluster motifs, I am unsure which PWM I should use.

Do you have any recommendations?

Thanks in advance.

- Laura

**Comment by SeppeDeWinter** [comment]
Hi @laurajohanne 

I'm not entirely sure what you are trying to do.
The metaclusters are already based on a motif similarity analysis (we used TomTom for this), motif belong to the same metacluster because they are similar.

If you would want to do this type of analysis again I would use each motif seperatly (so if a metacluster has two motifs, I would use both). But I'm not sure wether this is the thing you are trying to do.

All the best,

Seppe

**Comment by laurajohanne** [comment]
Okay, I see. Thank you very much!.I just wanted to compare the position weight matrixes between the metaclusters, but was unsure how the metaclusters where generated.

Best regards,
Laura

**Reply by SeppeDeWinter** [reply]
No worries! 

All the best,

Seppe
